src/style.css

```diff
@@ -1,3 +1,96 @@
+:root {
+  --blue-color: rgba(0, 172, 220, 1);
+}
+
+html,
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+  font-family: Roboto, serif;
+  font-weight: 400;
```

Comment:
You have specified only one font family. It's a good practice to specify a fallback font, in case the primary font can't be loaded. Please consider adding a fallback font.